hubble/src/arb.rs

```diff
@@ -141,31 +153,28 @@ impl Arb {
             .rollup_finalization_config
             .l1_next_node_num_slot_offset_bytes
             .inner() as usize;
-
         let raw_slot = self
             .l1_client
             .get_storage_at(
-                ethers::types::H160::from(self.rollup_finalization_config.l1_contract_address),
-                ethers::types::H256(
+                alloy::primitives::Address::new(
+                    (self.rollup_finalization_config.l1_contract_address).0,
+                ),
+                alloy::primitives::Uint::from_be_bytes(
                     self.rollup_finalization_config
                         .l1_latest_confirmed_slot
                         .to_be_bytes(),
                 ),
-                Some(ethers::types::BlockNumber::Number(l1_height.into()).into()),
             )
-            .await
-            .unwrap();
-
-        debug!(raw_slot = %H256::from(raw_slot));
+            .await?;
 
+        let raw_slot: B256 = raw_slot.into();
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/unionlabs/union/pull/2693#issuecomment-2293655644>#ef6b49</a><span> - 08/16/2024, 03:05 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The conversion of 'raw_slot' to 'B256' using '.into()' might not be explicit enough, leading to potential confusion or misuse. It is better to use 'B256::from(raw_slot)' for clarity.
</div>
<div id="fix">
<b>Fix:</b> Use 'B256::from(raw_slot)' instead of 'raw_slot.into()' to make the conversion explicit and clear.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -170,1 +170,1 @@
 -        let raw_slot: B256 = raw_slot.into();
 +        let raw_slot: B256 = B256::from(raw_slot);
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged